### PR TITLE
Switch FO parser to new data source

### PR DESCRIPTION
**Pull request by q--**
FO now also has solar, biomass and tidal generation (#2848). The old URL used by the parser doesn't have these, so I've switched  the parser to the new URL and updated it to handle these new generation types.

### CODE REVIEW

File: config/co2eq_parameters.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ exports.fossilFuelAccessor = (zoneKey, k, v) => {
   return (k == 'coal' ||
           k == 'gas' ||
           k == 'oil' ||
-          (k === 'unknown' && (zoneKey !== 'GB-ORK' && zoneKey !== 'UA' && zoneKey !== 'SG' && zoneKey !== 'PR')) ||
+          (k === 'unknown' && (zoneKey !== 'GB-ORK' && zoneKey !== 'UA' && zoneKey !== 'SG' && zoneKey !== 'PR' && zoneKey !== 'FO')) ||
```
Comment (FelixDQ) [inline config/co2eq_parameters.js:52]: We are actually in the progress of deprecating this file. So for now we also have to add this to the bottom of co2eq_parameters.json. (https://github.com/tmrowco/electricitymap-contrib/blob/9550a1ff35f2fad6e363c1afb096970f96fec687/config/co2eq_parameters.json#L2949-L3012)

**Review by FelixDQ — COMMENTED** [review]
Awesome. Only one comment. Otherwise this looks good to me :-)

File: config/co2eq_parameters.js (hunk 1 above)
Comment (q--) [reply config/co2eq_parameters.js:52]: Done

Comment (q--) [reply config/co2eq_parameters.js:52]: Also removed a now-unused import

**Review by jarek — APPROVED** [review]
:tada:

**Pull request merged**